js/src/dapps/registry/actions.js

```diff
@@ -1,8 +1,11 @@
 import registryAbi from './abi/registry.json';
 const { api } = window.parity;
-import * as lookup from './Lookup/actions.js';
+import * as accounts from './accounts/actions.js';
+import * as lookup from './lookup/actions.js';
+import * as events from './events/actions.js';
+import * as register from './register/actions.js';
 
-export { lookup };
+export { accounts, lookup, events, register };
 
 export const setContract = (contract) => ({ type: 'set contract', contract });
```

Comment:
But you don't need to add any libraries, I thought that we could just follow the format:
`{ type: 'set contract', payload: contract }`